src/cascadia/TerminalApp/TabHeaderControl.cpp

```diff
@@ -68,6 +75,18 @@ namespace winrt::TerminalApp::implementation
     void TabHeaderControl::RenameBoxLostFocusHandler(Windows::Foundation::IInspectable const& /*sender*/,
                                                      Windows::UI::Xaml::RoutedEventArgs const& /*e*/)
     {
+        // Log the data here, rather than in _CloseRenameBox. If we do it there,
+        // it'll get fired twice, once when the key is pressed to commit/cancel,
+        // and then again when the focus is lost
+
+        TraceLoggingWrite(
+            g_hTerminalAppProvider, // handle to TerminalApp tracelogging provider
+            "TabRenamerOpened",
```

Comment:
Shouldn't we provide some other event name?

Reply:
🤦‍♂️ yep, it sure should